mbzirc_ign/src/mbzirc_ign/bridges.py

```diff
@@ -169,3 +169,22 @@ def arm_camera_info(world_name, model_name, link_name):
         ign_type='ignition.msgs.CameraInfo',
         ros_type='sensor_msgs/msg/CameraInfo',
         direction=BridgeDirection.IGN_TO_ROS)
+
+def gripper_suction_contacts(model_name):
```

Comment:
hmm I think we probably don't want to bridge this topic. The contact data contains a scoped name of the collision that the gripper comes into contact with, which lets the user uniquely identify the object

We could either filter out the collision fields or just provide one force torque sensor to give some feedback that the suction gripper has pressed onto something.

Reply:
I have entirely removed the topic for now.